### ISSUE TITLE
Theme alternating entry colors?

### ISSUE THREAD

**Issue by Barbaross93** [philj56/tofi#63]
Hi there,

I was wondering if you would be interested in including options to specify the alternating entry foreground and background colors?
Personally, I find it super helpful in horizontal mode to distinguish between entries.

To give an example: https://en.opensuse.org/images/6/6e/Rofi-run-dialog.png

**Comment by philj56** [comment]
Yeah, there's no technical reason not to allow it - it should be a very simple feature to implement. I'll look into it.

**Comment by philj56** [comment]
Done in 3e11dc2327d70e860b3ad32db386aadd549393b3 - see my comment at https://github.com/philj56/tofi/issues/65#issuecomment-1322826949.
